.travis.yml

```diff
@@ -12,6 +12,7 @@ env:
  - IMAGE_PATH=wdqs-proxy/latest/
 
 script:
+ - bash ./$IMAGE_PATH/.travis/lint.sh
```

Comment:
It would be nice to have this run in prebuild just after check X.

This means the simple linting can be done before acquiring a bunch of vms to try building on.

Reply:
Sure, I've tweaked it so it runs all the lint.sh scripts in prebuild.

I've kept the lint.sh script separate because I think we'll want to put other linters than docker in there.